commons/zenoh-keyexpr/src/keyexpr_tree/traits/mod.rs

```diff
@@ -27,7 +27,7 @@ pub trait IKeyExprTree<'a, Weight> {
     /// Accesses the node at `key` if it exists, treating KEs as if they were completely verbatim keys.
     ///
     /// Returns `None` if `key` is not present in the KeTree.
-    fn node(&'a self, key: &keyexpr) -> Option<&Self::Node>;
+    fn node(&'a self, key: &keyexpr) -> Option<&'a Self::Node>;
```

Comment:
```suggestion
    fn node(&self, key: &keyexpr) -> Option<&Self::Node>;
```

This is equivalent to:

```suggestion
    fn node<'b>(&'b self, key: &keyexpr) -> Option<&'b Self::Node>;
```